atlas_commons/utils.py

```diff
@@ -83,6 +83,27 @@ def split_into_halves(
     return left_volume, right_volume
 
 
+def assign_hemispheres(annotation_path, halfway_offset: int = 0, hemispheres_path = None):
+    annotation = voxcell.VoxelData.load_nrrd(annotation_path)
```

Comment:
for making it easier to test, and easier to use, passing just a VoxcelData instance would be more convenient, I think.